rocksatdata.py

```diff
@@ -0,0 +1,7 @@
+import pandas as pd
```

Comment:
This file MUST be moved to an appropriate position in the file structure. This will likely go into the `ADXL1001/Test Data` folder. It should also be renamed to be more descriptive.